test/micro/org/openjdk/bench/vm/compiler/InterfaceCalls.java

```diff
@@ -124,6 +216,22 @@ public int testMonomorphic() {
         return as[0].getIntFirst();
     }
 
+    /** Tests single base interface method call */
+    @Benchmark
+    public void testIfaceCall(Blackhole bh) {
```

Comment:
Why these tests are not in the form of the other benchmarks? 

```
    @Benchmark
    public int test1stInt5Types() {
        FirstInterface ai = as[l];
        l = ++ l % asLength;
        return ai.getIntFirst();
    }
```

I suspect those are carefully written so that a single call-site would be used for all receivers, thus limiting the profile-guided optimizations.